tests/monotouch-test/GameplayKit/GKAgent3DTest.cs

```diff
@@ -35,6 +35,13 @@ namespace MonoTouchFixtures.GamePlayKit
 	[Preserve (AllMembers = true)]
 	public class GKAgent3DTest
 	{
+		[SetUp]
+		public void Setup ()
+		{
+			if (!TestRuntime.CheckSystemAndSDKVersion (9, 0))
```

Comment:
Any reason not to use `TestRuntime.CheckXcodeVersion (7, 0)`? It might be the same now, but not if watchOS 8.0 implements GameplayKit.

Reply:
good catch